src/Codeception/Template/Upgrade4.php

```diff
@@ -0,0 +1,137 @@
+<?php
+namespace Codeception\Template;
+
+use Codeception\Configuration;
+use Codeception\InitTemplate;
+
+class Upgrade4 extends InitTemplate
+{
+    const SURVEY_LINK = 'http://bit.ly/codecept-survey';
+    const DONATE_LINK = 'https://opencollective.com/codeception';
+
+    protected $modules = [
+        'WebDriver' => "codeception/module-webdriver",
+        'Yii2' => "codeception/module-yii2",
+        'Doctrine2' => "codeception/module-doctrine2",
+        'Asserts' => 'codeception/module-asserts',
+        'Filesystem' => 'codeception/module-filesystem',
+        'Cli' => 'codeception/module-cli',
+        'PhpBrowser' => 'codeception/module-phpbrowser',
+        'Db' => 'codeception/module-db',
+        'ZendExpressive' => 'codeception/module-zendexpressive',
+        'Symfony' => 'codeception/module-symfony',
+        'REST' => 'codeception/module-rest',
+        'Lumen' => 'codeception/module-lumen',
+        'Laravel5' => 'codeception/module-laravel5',
+        'Phalcon' => 'codeception/module-phalcon',
+        'ZF2' => 'codeception/module-zf2',
+        'Sequence' => 'codeception/module-sequence',
+        'SOAP' => 'codeception/module-soap',
+        'Redis' => 'codeception/module-redis',
+        'Queue' => 'codeception/module-queue',
+        'MongoDb' => 'codeception/module-mongodb',
+        'Memcache' => 'codeception/module-memcache',
+        'FTP' => 'codeception/module-ftp',
+        'DataFactory' => 'codeception/module-datafactory',
+        'Apc' => 'codeception/module-apc',
+        'AMQP' => 'codeception/module-amqp',
+
+    ];
+
+    public function setup()
+    {
+        if (!$this->isInstalled()) {
+            $this->sayWarning('Codeception is not installed in this dir.');
+            return;
+        }
+        $this->sayInfo('Welcome to Codeception v4 Upgrade wizard!');
+        $this->say('');
+        $this->say('Codeception is maintained since 2011, is free & open-source.');
+        $this->say('To make it better we need your feedback on it!');
+        $this->say('');
+        $this->say('Please take a minute and fill in a brief survey:');
+        $this->say('<bold>'  . self::SURVEY_LINK . '</bold>');
+        $this->say('');
+        $result = $this->ask('<question>Did you fill in the survey?</question>', true);
+        if ($result) {
+            $this->say('Thank you! ');
+        } else {
+            $this->say('Anyway...');
+        }
+        if (!file_exists('composer.json')) {
+            $this->sayWarning('Please use composer installation of Codeception');
+            throw new \Exception('composer.json not found, can\'t run upgrade');
```

Comment:
Output of exception probably looks ugly, sayWarning followed by return would be better.

Reply:
It looks good actually.